tslint.build-enforced.json

```diff
@@ -1,6 +1,7 @@
 {
     "rulesDirectory": ["tslint-microsoft-contrib"],
     "rules": {
+        "no-shadowed-variable": true,
```

Comment:
The current set of rules is alphabetized; could you put this in the same order?

Reply:
It's almost alphabetized but it's not (look for `function-constructor`). We still have a couple of rules to go through. I'll alphabetized when we're done. Sound good?

Reply:
I was waiting for the moment we moved all the rules to do the sorting... but we're already there, so I updated this PR removing the build enforced tslint config and alphabetizing the rules.